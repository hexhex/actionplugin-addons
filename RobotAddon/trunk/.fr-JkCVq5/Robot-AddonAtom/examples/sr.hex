objects(NO) :- &rdf["file:sr.ttl"]("<http://example.org/ex#file>",P,O),
		  &rdf[O](NS,NP,NO).

