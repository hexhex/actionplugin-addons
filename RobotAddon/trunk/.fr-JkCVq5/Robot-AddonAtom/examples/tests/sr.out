{objects("<file:sr.ttl>"), objects("'def'"), objects("'2'"), objects("'1'")}
